src/main/scala/mockws/FakeWSRequestHolder.scala

```diff
@@ -209,7 +218,24 @@ case class FakeWSRequestHolder(
   override def withBody[T](body: T)(implicit ev: BodyWritable[T]): Self =
     withBodyAndContentType(InMemoryBody(ev.transform(body)), ev.contentType)
 
-  override def withBody(file: java.io.File): Self = copy(body = FileBody(file))
+  override def withBody(file: java.io.File): Self = {
+    if (headers.contains("Content-Type")) {
+      copy(body = FileBody(file))
+    } else {
+      var contentTypeString = URLConnection.guessContentTypeFromName(file.getName)
```

Comment:
I am wondering if we are putting too much logic in the `MockWS` here.
Can we just simply let the user set the expected `Content-Type` header in the `Action`?